### ISSUE TITLE
Would be good to have a search bar on the site for searching by metadata

### ISSUE THREAD

**Issue by deepakunni3** [microbiomedata/pilot#17]
Would be good to have a search bar that can be used to search via keywords, controlled vocabularies and ontology terms.

**Comment by jeffbaumes** [comment]
@deepakunni3 I agree. My initial thought is a free-form search bar that searches all text fields for matches, including vocabularies and ontology terms. To search specific fields for specific values, I was envisioning an advanced search screen where the user can populate text boxes, drop-down menus, etc. Does this seem like a reasonable start?
